express-pipeline/src/test.ts

```diff
@@ -0,0 +1,66 @@
+// import pipeline from "./pipeline";
```

Comment:
Feel free to delete the old test files entirely. 

Where were you thinking of keeping the e2e tests? 
Maybe in some module like devtools?

Reply:
Yeah that's a good question. I think having e2e tests in its own folder. But there might be module specific tests that we'd want to keep in their relevant places. Testing is a weak-spot of mine, so if you have any thoughts I'd love to hear. Was also thinking this could be good to add to the repo: https://storybook.js.org/